packages/api/tests/lib/rules/test-rulesHelpers.js

```diff
@@ -835,15 +842,10 @@ test.serial('deleteRuleResources correctly deletes resources for kinesis rule',
 
 test.serial('deleteRuleResources correctly deletes resources for sns rule', async (t) => {
   const { testKnex } = t.context;
-  const lambdaStub = sinon.stub(awsServices, 'lambda')
-    .returns({
-      addPermission: () => ({
-        promise: () => Promise.resolve(),
-      }),
-      removePermission: () => ({
-        promise: () => Promise.resolve(),
-      }),
-    });
+  const lambdaMock = mockClient(awsServices.lambda());
+  lambdaMock.onAnyCommand().rejects();
+  lambdaMock.on(RemovePermissionCommand).resolves({});
+  lambdaMock.on(AddPermissionCommand).resolves({});
```

Comment:
nit: these used to resolve to undefined instead of an empty object.

Reply:
Fair - will address.

Reply:
1d3b0db02fc